src/drivers/config/drivers.yaml

```diff
@@ -27,5 +27,8 @@ version: "384.111"
 
 pre-installed-nvidia-path: /usr/local/nvidia
 
-
-skip-ib-installation: false
+# Azure VM builtin IB kernel modules into vmlinux image,
```

Comment:
It may not apply to Azure only. how about as below?

Some servers has already installed IB drivers, so this flag is disabled by default. If this flag is enabled, OpenPAI will try best to install the correct IB driver, but it may be failed due to compatibility.

Reply:
The installation works if ib drivers have already been installed, and we need to re-install drivers in `/var/drivers` path. The flag should be enabled in this case.
The only issue is linux kernel on Azure has builtin ib kernel modules, which fails the re-installation.

Reply:
We haven't tested on AWS, aliyun, and other various hardware configurations. So don't specify Azure here only, it may be a common case in cloud providers.